tools/aa/commands/InitCommand.java

```diff
@@ -95,11 +98,35 @@ public boolean run(String[] args) {
     return true;
   }
 
+  private boolean processArgs(String[] args) {
+    String[] initArgs = Arrays.copyOf(args, args.length);
+    boolean isAaAlreadySetUp = initArgs[0].equals("aa");
+    if (isAaAlreadySetUp) {
+      if (initArgs.length == 1) {
+        System.err.println(
+            RED_ERROR
+                + "Invalid usage. \n"
+                + "Please use \"aa init <base_path>\" command to init a base folder.");
+        return false;
+      }
+      // leave in `initArgs` array only `init` and `</path/to/base/folder>` arguments
+      initArgs = Arrays.copyOfRange(initArgs, 1, initArgs.length);
```

Comment:
Let's add:
// We remove the first element so that both calls to `aa init` have the same args.

I think once we add that, we can just reuse `args`. It's a reference anyway, so you're not changing the real array.

Reply:
Done.